mysqlLiquibaseResources/db/changelog/v1.0/01-changeset-project-table.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog
+        xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+                      http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.8.xsd">
+
+    <changeSet id="3" author="aktsay">
```

Comment:
Have we discussed versioning for changesets? Or is this `v1.0` as example? I believe it should be connected with app versions.

Reply:
Discussed and decided to keep simple consecutive numbers (1, 2, 3...)